lib/src/components/NxToggle/NxToggle.scss

```diff
@@ -7,12 +7,17 @@
 @import '../../scss-shared/nx-text-helpers';
 
 // In safari, the width: 0 and -moz-appearance settings fail to hide the <input>.
-// So instead, we use margin on the checkbox and corresponding negative margin on the toggle to hide
+// So instead, we use negative margin on the checkbox and offsetting positive margin on the toggle to hide
```

Comment:
I don't think this is up to date

Reply:
> I don't think this is up to date

Rewritten and moved here: https://github.com/sonatype/sonatype-react-shared-components/pull/410/commits/ca98f32a4ab06e790c0adeb13a3c8f83bc849e5b